lowtime/_lowtime_rs.pyi

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+import networkx as nx
+
+class PhillipsDessouky:
+    node_ids: list[int]
```

Comment:
Um what is this?

Reply:
Do you mean the existence of this stray variable or something else?
For now, I've removed the stray variable here. I think this was a typo because the member variables are not exposed to Python anyway (the pyo3 getters have been removed Rust-side as well).
If you meant something else, let me know. Otherwise, feel free to mark this as resolved.